app/util/address/index.test.ts

```diff
@@ -280,26 +301,38 @@ describe('isHardwareAccount,', () => {
     ).toBeFalsy();
   });
 });
-describe('getLabelTextByAddress,', () => {
+describe('getAccountLabelTextByKeyring,', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
   it('should return accounts.qr_hardware if account is a QR keyring', () => {
-    expect(
-      getLabelTextByAddress('0xB374Ca013934e498e5baD3409147F34E6c462389'),
-    ).toBe('accounts.qr_hardware');
+    expect(getAccountLabelTextByKeyring(mockQrKeyringAddress)).toBe(
+      'accounts.qr_hardware',
+    );
   });
 
   it('should return KeyringTypes.simple if address is a imported account', () => {
-    expect(
-      getLabelTextByAddress('0xd018538C87232FF95acbCe4870629b75640a78E7'),
-    ).toBe('accounts.imported');
+    expect(getAccountLabelTextByKeyring(mockSimpleKeyringAddress)).toBe(
+      'accounts.imported',
+    );
+  });
+
+  it('should return "Snaps (beta)" if account is a Snap keyring', () => {
```

Comment:
nit: I know the rest of the file is using the previous convention but that shouldn't stop us of using the new one

```suggestion
  it('returns "Snaps (beta)" if account is a Snap keyring', () => {
```